pages/cathocomponents.js

```diff
@@ -48,27 +48,27 @@ export default function CathoComponentsPage() {
                 />
               </article>
               <article>
-                <h5 className="TextFormattingH5">Logged Header Responsive</h5>
+                <h5 className="ComponentSubTitle">Logged Header Responsive</h5>
                 <HeaderLoggedResponsive
                   userProfile="STATUS_I_PROFISSIONAL"
                   hasSearch
                 />
-                <h5 className="TextFormattingH5">Simple Header</h5>
+                <h5 className="ComponentSubTitle">Simple Header</h5>
                 <HeaderSimple gtmPrefix="next" />
               </article>
             </article>
           </Container>
-          <Container className=" FlexContainer ColoredBackground Margins">
+          <Container className=" FlexContainer ColoredBackground">
             <Row>
               <Col
                 large={6}
                 medium={6}
                 small={4}
                 xmall={2}
-                className=" RoundedEdges WhiteBackground  PaddingLeftRight"
+                className="ComponentContainer WhiteBackground"
```

Comment:
```suggestion
                className="ComponentContainer"
```
The "ComponentContainer" class is always used together with the "WhiteBackground" class, what do you think about joining "WhiteBackground" to "ComponentContainer" to simplify the use of stylizations?

Reply:
There are two other specific use cases of the "WhiteBackground" class that you will need to adjust as well.

In the _app.js file the "WhiteBackground" class is used together with the "MainContent" class. In this case, you can also join it together to maintain a single class.

In the ChargeBar example, in quantum.js, there is a use of WhiteBackground in h6 that is not necessary. In that case you can remove it